.changeset/many-doors-prove.md

```diff
@@ -0,0 +1,5 @@
+---
+'@vanilla-extract/css': patch
+---
+
+Removed unused ahocorasick declaration reference
```

Comment:
This is describing the solution rather than the problem if fixes. Could be reworded to something like:
```suggestion
Fix a `tsc` error caused by a dependency's type declarations being unnecessarily bundled
```